### ISSUE TITLE
Review Bitrise and Screenshots build steps for l10n

### ISSUE THREAD

**Issue by flodolo** [mozilla-mobile/firefox-ios#11774]
cc @Delphine, @lmarceau, @isabelrios 

I looked at the code to check if shipping_locales.txt could be removed, and found references to yet another [external repository with scripts](https://github.com/mozilla-mobile/ios-l10n-scripts).
https://github.com/mozilla-mobile/firefox-ios/blob/main/bitrise.yml

All this seems very confusing, probably because different pieces have accumulated over time.

The workflow clones the l10n repository here, into a `firefoxios-l10n` folder
https://github.com/mozilla-mobile/firefox-ios/blob/main/bitrise.yml#L838-L839

Then runs `bootstrap.sh`, which removes that folder and makes another clone, same folder
https://github.com/mozilla-mobile/firefox-ios/blob/main/bootstrap.sh#L17-L18

In the screenshots tasks, the workflows then runs the `import-locales-firefox.sh` script from the external repo with the `--release` flag
https://github.com/mozilla-mobile/firefox-ios/blob/main/bitrise.yml#L915-L923

The [script in question](https://github.com/mozilla-mobile/ios-l10n-scripts/blob/master/import-locales-firefox.sh) makes yet another clone of `firefoxios-l10n`, but that's not really the problem here. 

I think running this is not needed at all, because translations are already imported weekly — and manually, potentially — via GitHub workflow. If you really want the latest strings for screenshots, then you should likely run bootstrap.sh [with the --importLocales](https://github.com/mozilla-mobile/firefox-ios/blob/main/bootstrap.sh#L27) flag. But I'm not sure you want screenshots to be different from the actual build.

Does this make sense?

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-4853)

**Comment by flodolo** [comment]
One more note: the import is [now importing](https://github.com/mozilla-mobile/LocalizationTools/pull/2) all locales available in the l10n repository.

**Comment by Delphine** [comment]
Hello, as I'm back on mobile now I'm trying to check-in on issues that were opened while I was away. Please let us know if you need any further info!

**Comment by Delphine** [comment]
@isabelrios Hello! Friendly reminder about this issue, as it seems to have fallen through the cracks! thank you

**Comment by isabelrios** [comment]
Finally I had sometime to look into this. My apologies for the delay.
I created the PR #13747 to address some of the comments here. I will try to explain deeper:

- `shiping_locales.txt` file was used by the L10nScreenshots tests and also when a new locale was added / removed so that it was correctly imported. For the first part we are fine with removing the file, for the second part, we would need confirmation from iOS devs.

- we need to clone `firefoxios-l10n` repo to have the .xliff files so that the tests can run for each locale. When bootstrap runs without arguments, that repo is not deleted or cloned again. We prefer to clone it and not run `bootstrap --importLocales` becuse that imples the string import which takes a considerable amount of time and we don't need the very latest strings to run the tests. 

- by running  .`/ios-l10n-scripts/import-locales-firefox.sh` we were using the old tooling for the l10n, but only to get the list of xliff, in the tests we are not importing / exporting. Anyway by removing this part we are not using that repo in any place now

I will ask for your review in the PR, hopefully that will address your comments / concerns...
